draft-ietf-webpush-encryption.md

```diff
@@ -279,32 +279,25 @@ application server MUST set the `rs` parameter in the `aes128gcm` content coding
 header to a size that is greater than the length of the plaintext, plus any
 padding (which is at least 2 octets).
 
-A push message MUST include a zero length `keyid` parameter in the content
-coding header.
+A push message MUST include the application server ECDH public key in the
+`keyid` parameter of the encrypted content coding header.
```

Comment:
It's ok - we can still extend/customize this format by adding info in headers :-)